R/ser.R

```diff
@@ -0,0 +1,180 @@
+# dXtX = attr(data$X, "d")
+# Xty = compute_Xty(data$X,R) we will compute this outside of SER and then plug in Xty into SER
+
+single_effect_regression =
```

Comment:
currently do we have two versions for single effect regression one using individual level data the other using sufficient statistics, or just one version using sufficient statistics? Because the computational complexity is different O(np) vs O(p^2), both versions may be useful for different scale of input data. We may not want to just unifiy them into sufficient stats with always O(p^2)?